scripts/install.sh

```diff
@@ -51,7 +51,7 @@ done
 OS=$(echo $OS | awk '{print tolower($0)}')
 
 # Get the latest version of swctl.
-VERSION=$(curl "https://raw.githubusercontent.com/apache/skywalking-website/master/data/releases.yml" | grep --after-context=7 "name: SkyWalking CLI" | grep "version" | grep -o "[0-9].[0-9].[0-9]")
+VERSION=$(curl "https://raw.githubusercontent.com/apache/skywalking-website/master/data/releases.yml" | grep --after-context=7 "name: SkyWalking CLI" | grep "version" | grep -Eo "[0-9].[0-9]+.[0-9]")
```

Comment:
You might be using GNU grep so it supports regex by default, but `-E` is needed in non-GNU grep. What is the problem in your case?

Reply:
I can't install skywalking-cli on Centos 7.x，the match result is None，i need to modify the install.sh regex to install skywalking-cli v0.10.0

Reply:
> I can't install skywalking-cli on Centos 7.x，the match result is None，i need to modify the install.sh regex to install skywalking-cli v0.10.0

OK sorry I missed one point in your changed file, `"[0-9].[0-9].[0-9]"` --> `"[0-9].[0-9]+.[0-9]"` is totally right, I meant modification from `-o` to `-Eo` is not necessary ?

Reply:
Also I'd suggest just write the regex as `"[0-9]+.[0-9]+.[0-9]+"`, this is future-proof so that we won't be error if there is versions like `11.11.11`

Reply:
it's right, i just point out one of the problems. and here you need to specify `-E` on CentOS，it‘s my test result

Reply:
@Alexxxing Thanks!!